public/app/plugins/datasource/tempo/datasource.ts

```diff
@@ -35,6 +35,9 @@ export interface TempoJsonData extends DataSourceJsonData {
   serviceMap?: {
     datasourceUid?: string;
   };
+  search?: {
+    hide?: boolean;
```

Comment:
Out of curiosity, why do we need the option to hide the search tab? Is it because we're planning to make it selected by default and users without switch would need to change it for every query?

Reply:
This was requested by the Tempo team. There are 5 tabs right now in Tempo which is getting a bit crowded, so this is one way to reduce that. 

Because search is relatively 'expensive' to operate on the backend, it's opt-in so many users will not enable search on the backend, so in that case, they can hide it on the frontend.